package.json

```diff
@@ -4,20 +4,19 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "npm run remove-dist && npm run build:production && npm run build:static",
-    "build:production": "webpack -p --mode=production",
-    "build:test": "webpack -p --mode=production --include-tests",
-    "build:dev": "npm run remove-dist && npm run build:static && webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo",
+    "build": "npm run build:production && npm run build:static",
+    "build:production": "webpack --mode=production",
+    "build:test": "webpack --mode=production --include-tests",
```

Comment:
Arguably, these can use `npm run build:production` as their baseline, eg:
```suggestion
    "build:test": "build:production -- --include-tests",
```
I dunno if that's preferred by others—it's a bit more DRY, so if you want to commit these suggestions feel free without it needing another review. If not, that's cool too.